packages/contracts/test/TestUtils.t.sol

```diff
@@ -70,6 +70,15 @@ import {
 import { SafeL2 as SafeL2_1_4_1 } from "@gnosis.pm/safe-contracts-1.4.1/SafeL2.sol";
 import { Safe as Safe_1_4_1 } from "@gnosis.pm/safe-contracts-1.4.1/Safe.sol";
 
+struct SphinxTransaction {
+    address to;
+    uint256 value;
+    bytes txData;
+    IEnum.GnosisSafeOperation operation;
+    uint256 gas;
+    bool requireSuccess;
+}
+
```

Comment:
Moved this from `SphinxPluginTypes.sol` because we now only use it in tests and not in the plugin. (It was previously used in `Sphinx.sol:sphinxEstimateMerkleLeafGas`. We now use the `GnosisSafeTransaction` struct instead because it contains fewer fields. Was necessary to merge the collection and gas estimation logic)